Makefile

```diff
@@ -39,25 +38,13 @@ logs: ## Shows logs of all running projects' containers
 logs-recent: ## For CI
 	@ make -C docker logs-recent
 
-.PHONY: bash
```

Comment:
These are already captured in `stubs.mk` and don't make since to run from the top level directory anywho, just cleaning and reducing lines.